contracts/modules/governance/NXMaster.sol

```diff
@@ -13,32 +13,35 @@ import "../../interfaces/IQuotation.sol";
 import "../../interfaces/IQuotationData.sol";
 import "../../interfaces/ITokenController.sol";
 import "../../interfaces/ITokenData.sol";
-import "../capital/LegacyPoolData.sol";
 import "./external/Governed.sol";
 import "./external/OwnedUpgradeabilityProxy.sol";
 
 contract NXMaster is INXMMaster, Governed {
   using SafeMath for uint;
 
-  struct EmergencyPause {
-    bool pause;
-    uint time;
-    bytes4 by;
-  }
-
-  EmergencyPause[] public emergencyPaused;
+  uint public _unused0;
 
-  bytes2[] internal allContractNames;
+  bytes2[] public contractCodes;
   mapping(address => bool) public contractsActive;
-  mapping(bytes2 => address payable) internal allContractVersions;
+  mapping(bytes2 => address payable) public contractAddresses;
   mapping(bytes2 => bool) public isProxy;
-  mapping(bytes2 => bool) public isUpgradable;
+  mapping(bytes2 => bool) public isReplaceable;
 
   address public tokenAddress;
   bool internal reentrancyLock;
   bool public masterInitialized;
   address public owner;
-  uint public pauseTime;
+  uint public _unused1;
```

Comment:
You can initialize _emergencyAdmin_ and _paused_ in the constructor and mark the remaining unused space from the slot as uint88 _unused which you can also initialize to 0.

Reply:
This is a proxy. Storage variable initializations don't have any effect in constructor (but it works with immutables).

Reply:
s/constructor/initializer